source/_integrations/workday.markdown

```diff
@@ -146,6 +146,20 @@ binary_sensor:
       - 'Christmas Day'
 ```
 
+This example has two workday sensors. The default mon-fri workday and a 2nd `schoolday-name` sensor for tue-thu.
+
+```yaml
+# Example 5 configuration.yaml entry
+binary_sensor:
+  - platform: workday
+    name: workday-default
```

Comment:
Enquote strings please, and let's use a descriptive name. This applies to all string in this YAML example. For more information see:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

Also, using descriptive names, makes it more readable and less needed to explain what it is in the above text preceding this example.